apps/kg/kvclient.py

```diff
@@ -35,46 +32,34 @@ def __init__(self):
                           help='dataset name, under data_path')
         self.add_argument('--format', type=str, default='1',
                           help='the format of the dataset.')
-        self.add_argument('--save_path', type=str, default='ckpts',
+        self.add_argument('--save_path', type=str, default='../ckpts',
```

Comment:
why the default path is in the parent directory?

Reply:
Because the distributed task runs on sub-directory.